lib/schemas/prior_authority/v1.json

```diff
@@ -698,7 +698,7 @@
       "x-pii": false,
       "type": ["string", "null"],
       "title": "Stashed provisional decision",
-      "enum": ["granted", "part_grant", "rejected"],
+      "enum": ["granted", "part_grant", "rejected", null],
```

Comment:
Worth adding a comment to the README or something to indicate we need to include this